fpm-alpine/Dockerfile

```diff
@@ -17,15 +17,12 @@ ONBUILD LABEL org.label-schema.vendor="Wizbii" \
       org.label-schema.schema-version="1.0" \
       org.label-schema.build-date="${BUILD_DATE}"
 
-RUN apk add --no-cache autoconf openssl-dev g++ make pcre-dev icu-dev zlib-dev libzip-dev && \
+RUN apk add --no-cache gnu-libiconv autoconf openssl-dev g++ make pcre-dev icu-dev zlib-dev libzip-dev && \
     mv "$PHP_INI_DIR/php.ini-production" "$PHP_INI_DIR/php.ini" && \
     pecl install mongodb && \
     docker-php-ext-enable mongodb && \
     docker-php-ext-install bcmath intl opcache zip sockets && \
     version=$(php -r "echo PHP_MAJOR_VERSION.PHP_MINOR_VERSION;") && \
-    curl -A "Docker" -o $(php -r "echo ini_get('extension_dir');")/blackfire.so -D -s https://packages.blackfire.io/binaries/blackfire-php/1.24.4/blackfire-php-alpine_amd64-php-$version.so && \
-    printf "extension=blackfire.so\nblackfire.agent_socket=tcp://10.1.1.1:8707\n" > $PHP_INI_DIR/conf.d/blackfire.ini && \
-    apk add gnu-libiconv --update-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/community/ --allow-untrusted && \
```

Comment:
libiconv is now in the stable repository for alpine 3.10 ✨

Reply:
Nice